lib/db.js

```diff
@@ -476,43 +467,43 @@ DB.prototype._createTable = function (keyspace, schema, tableName) {
     });
 
     // Finally, create the main data table
-    var cql = 'create table if not exists '
+    var sql = 'create table if not exists '
         + dbu.cassID(keyspace + '_' + tableName) + ' (';
     for (var attr in schema.attributes) {
         var type = schema.attributes[attr];
-        cql += dbu.cassID(attr) + ' ';
+        sql += dbu.cassID(attr) + ' ';
         switch (type) {
```

Comment:
Most of the types enumerated in this switch do not exist in SQLite. Please consult [the documentation](https://www.sqlite.org/datatype3.html) and make the appropriate changes.